lib/jekyll-linkpreview.rb

```diff
@@ -122,7 +96,7 @@ def get_properties(url)
 
       private
       def get_url_from(context)
-        context.scopes[0].key?(@markup) ? context.scopes[0][@markup] : @markup
+        context[@markup]
```

Comment:
Becomes much simpler!
Now users have to pass quoted url like `{% linkpreview 'https://github.com' %}`, but it may be acceptable.